compose.py

```diff
@@ -71,12 +71,12 @@ def start_dockerfile(ddir, images, argmap, dockerfile_name, backends):
         argmap["TRITON_VERSION"], argmap["TRITON_CONTAINER_VERSION"], images["full"]
     )
 
-    # PyTorch, TensorFlow 1 and TensorFlow 2 backends need extra CUDA and other
+    # PyTorch, TensorFlow backends need extra CUDA and other
     # dependencies during runtime that are missing in the CPU-only base container.
     # These dependencies must be copied from the Triton Min image.
     if not FLAGS.enable_gpu and (
         ("pytorch" in backends)
-        or ("tensorflow1" in backends)
+        or ("tensorflow" in backends)
         or ("tensorflow2" in backends)
```

Comment:
do we need the `tensorflow2` case as well?

Reply:
We do it for [build.py](https://github.com/triton-inference-server/server/blob/main/build.py#L2484) but I think yeah makes sense to remove